doc/js_tutorials/js_assets/js_dnn_example_helper.js

```diff
@@ -0,0 +1,141 @@
+getBlodFromImage = function(inputSize, mean, std, swapRB, canvasId) {
+    let mat = cv.imread(canvasId);
+    let matC3 = new cv.Mat(mat.matSize[0], mat.matSize[1], cv.CV_8UC3);
+    cv.cvtColor(mat, matC3, cv.COLOR_RGBA2RGB);
+    let input = cv.blobFromImage(matC3, std, new cv.Size(inputSize[0], inputSize[1]),
+                                 new cv.Scalar(mean[0], mean[1], mean[2]), swapRB);
+    mat.delete();
+    matC3.delete();
+    return input;
+}
+
+getBlodFromFrame = function(inputSize, mean, std, swapRB, frame) {
+    let matC3 = new cv.Mat(frame.matSize[0],frame.matSize[1],cv.CV_8UC3);
+    cv.cvtColor(frame, matC3, cv.COLOR_RGBA2RGB);
+    let input = cv.blobFromImage(matC3, std, new cv.Size(inputSize[0], inputSize[1]),
+                                 new cv.Scalar(mean[0], mean[1], mean[2]), swapRB);
+    matC3.delete();
+    return input;
+}
```

Comment:
Can you check in runtime if object is instance of `cv.Mat` (frame) or a canvas? So create a single function